metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/themes/metron_theme.json

```diff
@@ -214,6 +214,73 @@
             }
           },
           {
+            "name": "profiler",
```

Comment:
I noticed the layout is slightly different in the Ambari UI than other tabs (Enrichment for example).  I don't think it necessarily looks bad (might even be an improvement) but ideally all the tabs are consistent.

Reply:
Yeah, I agree, we should have a consistent look and feel here.  I'll adjust.  I think the REST panel looks like this and I cut and pasted.  I'll look into it.